addons/stock_account/models/account_chart_template.py

```diff
@@ -31,5 +31,13 @@ def generate_properties(self, acc_template_ref, company, property_list=None):
             account = self[field]
             value = acc_template_ref[account].id if account else False
             PropertyObj._set_default(field, "product.category", value, company)
-
+        if self.property_stock_production_account_id:
+            production_account_id = acc_template_ref[self.property_stock_production_account_id].id if account else False
```

Comment:
```suggestion
            production_account_id = acc_template_ref[self.property_stock_production_account_id].id
```
`account` is defined on the previous loop. You should check the existence of `self.property_stock_production_account_id` instead. But since you check it in the previous `if` statement, this condition is useless.

Reply:
yes, right, removing.....

Reply:
@viti-odoo remove it and push a separate commit for this